pkg/api/handlers/compat/containers_pause.go

```diff
@@ -2,27 +2,50 @@ package compat
 
 import (
 	"net/http"
+	"os"
 
 	"github.com/containers/podman/v2/libpod"
+	"github.com/containers/podman/v2/libpod/define"
 	"github.com/containers/podman/v2/pkg/api/handlers/utils"
+	"github.com/containers/podman/v2/pkg/domain/entities"
+	"github.com/containers/podman/v2/pkg/domain/infra/abi"
+	"github.com/gorilla/schema"
+	"github.com/pkg/errors"
 )
 
 func PauseContainer(w http.ResponseWriter, r *http.Request) {
+	decoder := r.Context().Value("decoder").(*schema.Decoder)
 	runtime := r.Context().Value("runtime").(*libpod.Runtime)
+	containerEngine := abi.ContainerEngine{Libpod: runtime}
+	query := struct {
+		All bool `schema:"all"`
```

Comment:
We can't do this without changing return type. It is an *absolutely terrible* user experience if we have All but do not return every error.

Reply:
Yes I agree, and I am running out of time to fix the returns.